examples/practice-typescript/src/components/SideBarMenu/SideBarMenu.tsx

```diff
@@ -0,0 +1,28 @@
+import SideBarMenuItem, {
+  SideBarMenuItemProps,
+} from 'components/SideBarMenuItem/SideBarMenuItem';
+import './sideBarMenu.scss';
+
+interface SideBarMenuProps {
+  sideBarMenuItemProps: SideBarMenuItemProps[];
+}
+
+const SideBarMenu = ({ sideBarMenuItemProps }: SideBarMenuProps) => {
+  return (
+    <ul className="aside__list">
+      {sideBarMenuItemProps.map((item) => (
```

Comment:
I don't understand this line `{sideBarMenuItemProps.map((item)=>`
Why the props should have item as it children?

Reply:
Fixed